### PR TITLE
Add Ipmath filter plugin

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/filter/ipmath.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    filter_map = {
+        # IP addresses and networks
+        "ipmath": _ipmath
+    }
+
+    def filters(self):
+        if netaddr:
+            return self.filter_map
+        else:
+            # Need to install python's netaddr for these filters to work
```
Comment (KB-perByte) [inline plugins/filter/ipmath.py:183]: can we update comments lingo to be more of a dev doc than generic statements, I think that makes the public code more formal?

**Comment by ashwini-mhatre** [comment]
recheck

File: plugins/filter/ipmath.py (hunk 1 above)
Comment (ashwini-mhatre) [reply plugins/filter/ipmath.py]: ack

Comment (ashwini-mhatre) [reply plugins/filter/ipmath.py]: ack

Comment (ashwini-mhatre) [reply plugins/filter/ipmath.py]: added this "Need to install python's netaddr for these filters to work" in _need_netaddr function https://github.com/ansible-collections/ansible.utils/pull/115/commits/3e186211a3ef40653917acabd33cf8e1b29da737

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**